server/data/data-library/src/test/java/org/apache/james/user/lib/model/DefaultUserTest.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.james.user.lib.model;
+
```

Comment:
Top apachev2 license missing. That's compulsory.

Reply:
Oops, nice catch. I will add it.